src/bino.hpp

```diff
@@ -118,7 +121,8 @@ class DirectOFile {
  */
 class UnformattedOFile {
```

Comment:
Binary flag is gone. See #13

Reply:
I apologize--typical Git mistake--I merged `main` but not `origin/main` into this branch (a while ago). This is fixed now.